spec/classes/district_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require_relative '../../lib/turkish_cities/district'
+
+RSpec.describe District do
+  describe '#list_neighborhoods' do
+    context 'when input is supported' do
+      it 'finds neighborhoods of given city and district' do
+        expect(District.new('Artvin', 'Yusufeli').neighborhoods(nil).count).to eq 674
+        expect(District.new('İstanbul', 'Adalar').neighborhoods(nil).count).to eq 5
+        expect(District.new('Ankara', 'Mamak').neighborhoods(nil).count).to eq 64
```

Comment:
Maybe you can add the problematic case you encountered before
```
TurkishCities.list_neighborhoods("Izmir", "Konak")

District.new('İzmir', 'Konak').neighborhoods(nil)
```

Reply:
of course 👍